tests/testdata/simple_tags/index.ts

```diff
@@ -1,9 +1,5 @@
-// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+// Copyright 2016-2023, Pulumi Corporation.  All rights reserved.
```

Comment:
Did you meant to include the changes to the `index.ts` and `resource.ts` files?  They seem unrelated to what you're describing, but maybe I'm still missing something :)

Reply:
Yeh was just simplifying this test, it was copy pasted from another test when first made but it really doesn't need the resource creation in it. Plus all the package names didn't match the test name, doesn't really matter, but tidyness.